cves/kernel/CVE-2016-2782.yml

```diff
@@ -75,7 +75,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1312670, 1283374]
```

Comment:
You might be able to put a description of these bugs as well in a note, so someone will understand what kind they are instead of just numbers.